README.md

```diff
@@ -279,6 +292,22 @@ Kafka Cluster Connection Details (`kafka-lag-exporter.clusters[]`)
 | `admin-client-properties` | `{}`        | No       | A map of key value pairs used to configure the `AdminClient`. See the [Admin Config](https://kafka.apache.org/documentation/#adminclientconfigs) section of the Kafka documentation for options.          |
 | `labels`                  | `{}`        | No       | A map of key value pairs will be set as additional custom labels per cluster for all the metrics in prometheus.                                                                                           |
 
+Redis Details (`kafka-lag-exporter.redis{}`)
+
+| Key          | Default                | Required | Description                                                                                                         |
+|--------------|------------------------|----------|---------------------------------------------------------------------------------------------------------------------|
+| `enabled`    | `"false"`              | No       | Switch to enable or disable the Redis table (if enabled, the memory tables will not be created).                    |
```

Comment:
```suggestion
| `enabled`    | `false`                | No       | Switch to enable or disable the Redis table (if enabled, the memory tables will not be created).                    |
```

Double quotes shouldn't be necessary for any of the defaults in this new section.

Reply:
> (if enabled, the memory tables will not be created).

Did you mean to say "if disabled"? When are the tables created?